index.js

```diff
@@ -74,12 +82,14 @@ module.exports = {
             page_out_of_boundaries: 'Trang số %{page} đã vượt giới hạn',
             page_out_from_end: 'Không thể đi tiếp sau trang cuối',
             page_out_from_begin: 'Không thể trở lại trước trang 1',
+            page_rows_per_page: 'Hàng trên mỗi trang',
```

Comment:
I think 'Rows per page:' will be 'Số hàng trên mỗi trang:'. you also missed colon.